packages/unlock-js/src/subgraph/schema.graphql

```diff
@@ -25,6 +26,26 @@ query allLocks(
     maxKeysPerAddress
     createdAtBlock
     totalKeys
+    referrerFees(skip: $referrerFeeSkip) {
+      id
+      referrer
+      fee
+      lock {
+        id
+        address
+        name
+        expirationDuration
+        tokenAddress
+        symbol
+        price
+        lockManagers
+        version
+        maxNumberOfKeys
+        maxKeysPerAddress
+        createdAtBlock
+        totalKeys
+      }
```

Comment:
I am not sure we need to add the lock details here again since `referrerFees` is already _inside_ the `lock` object.